test/unit/LivepeerETHTicketBroker.js

```diff
@@ -243,4 +245,19 @@ contract("LivepeerETHTicketBroker", accounts => {
             })
         })
     })
+
+    describe("setSignerRevocationPeriod", () => {
+        it("reverts when called by an account that is not the owner", async () => {
```

Comment:
Small nit: We're really checking that the sender is the Controller's owner rather than the owner of LivepeerETHTicketBroker

Reply:
👍